src/app/utils/cn.js

```diff
@@ -0,0 +1,6 @@
+import { clsx } from 'clsx';
+import { configuredTwMerge } from './configuredTwMerge';
+
+export default function cn(...inputs) {
```

Comment:
🔴 please do not use default export, that will allow to import it like `import { cn } from '@/app/utils'`

Reply:
🟠  and meaningful name like `classes` would be nice here

Reply:
Thanks for comment! Done.